permissions/plugin-azure-cli.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "azure-cli"
+paths:
+- "org/jenkins-ci/plugins/azure-cli"
```

Comment:
This seems wrong: https://github.com/jenkinsci/azure-cli-plugin/blob/master/pom.xml#L11

Please see the checklist for a reference to the docs in this repo on how this works.